<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editor_add_artist</class>
 <widget class="QDialog" name="editor_add_artist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>249</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserir Artista (musician or compositor)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>101</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rb_musician">
      <property name="text">
       <string>Musician</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rd_compositor">
      <property name="text">
       <string>Compositor</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>330</y>
     <width>156</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lb_fotografia">
      <property name="text">
       <string>Fotografia:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_upload_photo">
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>207</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="lb_instrument">
      <property name="text">
       <string>Instrument:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ln_instrument"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>34</y>
     <width>176</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="lb_name">
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ln_name"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>65</y>
     <width>184</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="lb_surname">
      <property name="text">
       <string>Apelido:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ln_surname"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>96</y>
     <width>248</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="lb_date">
      <property name="text">
       <string>Data de Nascimento:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dt_birthdate"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>129</y>
     <width>324</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="lb_biografia">
      <property name="text">
       <string>Biografia:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="ln_biografia"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>168</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QPushButton" name="bt_close">
      <property name="text">
       <string>Fechar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_add_artist">
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bt_close</sender>
   <signal>clicked()</signal>
   <receiver>editor_add_artist</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
